docs/guides/forms/Forms.stories.mdx

```diff
@@ -1,5 +1,79 @@
-import { Meta } from "@storybook/addon-docs";
+import { Canvas, Meta, Story } from "@storybook/addon-docs";
+import * as FormSamples from "./FormsSamples";
 
-<Meta title="Guides/Forms/Main" />
+<Meta title="Guides/Forms/Guide" />
 
 # Forms
+
+Web forms play a crucial role in web applications, as they are the primary form to collect user information.
+By providing a standardised way to capture user input and validat it, we can ensure a consistent user experience across the different applications.
+
+## Form Components
+
+UI Kit provides a set of **Form Components** to build forms. These components have the goals of providing a **consistent React API** to build forms **following the HV Design spec** and
+**ease of use and integration with** native HTML form, React forms and form libraries.
+
+The base API of all the form components is established by the **[Form Element](https://lumada-design.github.io/uikit/master/?path=/docs/guides-forms-form-element--docs)** component.
+It provides the necessary props to listen to the events, manage the component `status` as well as a `value` prop to make each component **controlled**.
+
+The UI Kit library currently offers the following form components:
+
+- [Checkbox](./?path=/docs/components-checkbox-checkbox--docs) & [Checkbox Group](./?path=/docs/components-checkbox-checkbox-group--docs)
+- [Date Picker](./?path=/docs/components-date-picker--docs)
+- [Dropdown](./?path=/docs/components-dropdown--docs)
+- [Input](./?path=/docs/components-input--docs)
+- [Radio Button](./?path=/docs/components-radio-radio--docs) & [Radio Button Group](./?path=/docs/components-radio-radio-group--docs)
+- [Search Box](./?path=/docs/components-input-search-box--docs)
+- [Selection List](./?path=/docs/components-list-selection-list--docs)
+- [Slider](./?path=/docs/components-slider--docs)
+- [Switch](./?path=/docs/components-switch--docs)
+- [Tags Input](./?path=/docs/components-tag-tags-input--docs)
+- [Text Area](./?path=/docs/components-text-area--docs)
+- [Time Picker](./?path=/docs/components-time-picker--docs)
+
+## Building forms
+
+There are multiple ways to build forms in React, and each one has its pros and cons. In the following section we will cover the most common ways to build forms in React with UI Kit.
+
+### Native form
+
+The simplest way to build a form is by using a native `form`, using the [built-in form validation](https://developer.mozilla.org/en-US/docs/Learn/Forms/Form_validation#using_built-in_form_validation)
+and reading the results from `FormData`.
+It's a good option for **simple forms** using form components based on native form controls (eg. `input`) where **custom validation is not needed**.
+
+**Note:** Not all UI Kit form components integrate with the native form data or validation.
+
+<Canvas>
+  <Story name="Native Form" story={FormSamples.NativeFormStory} />
+</Canvas>
+
+### React form
```

Comment:
Maybe we could make the text field and dropdown fields required for the native, react, react hook, and formik forms just to showcase some error messages